### PR TITLE
Model v4

### PR DESCRIPTION
**Pull request by sakshamsoni97**
Merging model_V4 code. Maine changes:
- able to start fitting from a different date
- running scripts using a run config file instead of argument list

### CODE REVIEW

File: DELPHI_utils_V3_new_wave.py [hunk 1]
```diff
@@ -0,0 +1,1340 @@
+import os
```
Comment (MichaelLLi) [inline DELPHI_utils_V3_new_wave.py:1]: Can you delete this file? I think this is the hacky code right?

File: DELPHI_compare.py [hunk 2]
```diff
@@ -118,32 +112,26 @@
...
-            if annealing_available:
+            if not annealing_select:
                 logger.warning(f'Annealing performs worse in {country} - {province}')
+                global_parameters_best = global_parameters_best.append(tnc_params, ignore_index=True)
             else:
-                logger.warning(f'Annealing parameters not present for {country} - {province}')
-            global_parameters_best = global_parameters_best.append(tnc_params, ignore_index=True)
-        else:
-            global_parameters_best = global_parameters_best.append(annealing_params, ignore_index=True)
+                global_parameters_best = global_parameters_best.append(annealing_params, ignore_index=True)
```
Comment (MichaelLLi) [inline DELPHI_compare.py:132]: I found it is better/easier to just overwrite the latest params file than creating a separate one - because otherwise we need to manually update it again - can you revert the changes on this? Also we need a V4 equivalent right?

Comment (MichaelLLi) [reply DELPHI_compare.py:132]: or update this to be V4

**Review by MichaelLLi — CHANGES_REQUESTED** [review]

File: DELPHI_utils_V3_new_wave.py (hunk 1 above)
Comment (sakshamsoni97) [reply DELPHI_utils_V3_new_wave.py:1]: Right! I didn't realize it got pushed

File: DELPHI_compare.py (hunk 2 above)
Comment (sakshamsoni97) [reply DELPHI_compare.py:132]: I'll write a separate script for V4 because I'll need to modify a few things for V4 in this. For this one I'll revert the change.
